content/docs/dataform-web/how-to-guides/environments-example.md

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
IMO, you don't need this page, just make the existing one better?

We have lots of pages where we have a description of the thing then some examples. I think it's simpler if we have all the environments stuff in one page.

Reply:
Sure, works well